xml/Microsoft.Extensions.Primitives/StringValues.xml

```diff
@@ -1409,9 +1445,10 @@
         <Parameter Name="item" Type="System.String" />
       </Parameters>
       <Docs>
-        <param name="item">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="item">The <see cref="T:System.String" /> to locate in the <see cref="T:Microsoft.Extensions.Primitives.StringValues" />.</param>
+        <summary>Determines whether a string is in the <see cref="T:Microsoft.Extensions.Primitives.StringValues" />.</summary>
+        <returns>
+          <see langword="true" /> if <paramref name="item">item</paramref> is found in the <see cref="T:Microsoft.Extensions.Primitives.StringValues" />; otherwise, <see langword="false" />.</returns>
```

Comment:
Strange. the paramref format is wrong:

```suggestion
          <see langword="true" /> if <paramref name="item" /> is found in the <see cref="T:Microsoft.Extensions.Primitives.StringValues" />; otherwise, <see langword="false" />.</returns>
```